tests/filter_task_by_tag.yml

```diff
@@ -0,0 +1,56 @@
+name: Filter Task by Tag Job
+description: Job to test the filter task by tag feature
+service: TES
+versions:
+  - 1.1.0
+tags:
+  - logical
+  - filter
+  - filter task by tag test
+  - all
+jobs:
+  - name: create_task
+    description: Create a new TES task
+    endpoint: /tasks
+    operation: POST
+    request_body: |
+      {
+        "name": "ComplianceTest",     
+        "description": "Compliance Test",
+        "executors": [
+          {
+            "image": "alpine",
+            "command": [
+              "echo",
+              "hello"
+            ]
+          }
+        ],
+        "tags": {
+          "foo": "bar",
+          "baz": "bat",
+          "lorem": "ipsum",
+          "task": "create"
+        }
+      }
+    response:
+      200:
+  - name: list_tasks
+    description: Retrieve the list of tasks tracked by the TES server
```

Comment:
I think this test should be a lot more extensive to capture all of the behaviors of the tag filters. That is, submit more tasks with different tags and apply filters in different `GET /tasks` calls.

Reply:
Yes, I will add them. They can be added to the same test only.